scripts/components/RepairRequestForm.jsx

```diff
@@ -73,32 +77,35 @@ var RepairRequestForm = React.createClass({
   render() {
     var { description, dataUri } = this.state;
     var errorMessage;
+    var standardActions = [
+      { text: 'Cancel' },
+      { text: 'Submit', onTouchTap: this.onSubmit, ref: 'submit' }
+    ];
     return (
       <div style={style.formContainer}>
-        <Paper zDepth={1}>
-          <form style={style.form} onSubmit={this.onSubmit}>
-            <h3 style={style.heading}>Lodge a New Repair Request</h3>
-            <div style={style.error}> { errorMessage } </div>
-            <TextField
-              value={description}
-              multiLine={true}
-              name="Description"
-              onChange={this.onChange.bind(this, 'description')}
-              floatingLabelText="Description" />
-            <div style={style.inputContainer}>
-              {dataUri ?
-                (<img style={style.img} src={this.state.dataUri} />) :
-                (null)}
-              <input type="file" name="file" onChange={this.onFileSelected} />
-            </div>
-            <RaisedButton
-              type="submit"
-              label="Lodge Repair Request"
-              primary={true}
-              backgroundColor="#2ECC71"
-              style={style.button} />
-          </form>
-        </Paper>
+        <RaisedButton label="Lodge a New Repair Request"
+                      primary={true}
+                      onClick={this.onButtonClick} />
+        <Dialog
+          title="Lodge a New Repair Request"
+          actions={standardActions}
+          actionFocus="submit"
+          modal={this.state.modal}
+          ref="dialog">
+          <div style={style.error}> { errorMessage } </div>
+          <TextField
+            value={description}
+            multiLine={true}
+            name="Description"
+            onChange={this.onChange.bind(this, 'description')}
+            floatingLabelText="Description" />
+          <div style={style.inputContainer}>
```

Comment:
Update this to use the `<ImageSelector>` component I made. I mentioned this to the team in the chat the other day. This component supports things like specifying a maximum file size allowed, and will allow for us to add features to the component if necessary. We need to have an image upload control in tonnes of places, so it will be easier to maintain if they all are using a single component.